governance/default_test.go

```diff
@@ -277,39 +285,33 @@ func TestGovernance_ClearVotes(t *testing.T) {
 }
 
 func TestGovernance_GetEncodedVote(t *testing.T) {
-	var err error
 	gov := getGovernance()
 
 	for _, val := range goodVotes {
 		_ = gov.AddVote(val.k, val.v)
 	}
 
 	l := gov.voteMap.Size()
-	for i := 0; i > l; i++ {
+	for i := 0; i < l; i++ {
 		voteData := gov.GetEncodedVote(common.HexToAddress("0x1234567890123456789012345678901234567890"), 1000)
 		v := new(GovernanceVote)
 		rlp.DecodeBytes(voteData, &v)
 
-		if v, err = gov.ParseVoteValue(v); err != nil {
-			assert.Equal(t, nil, err)
-		}
-
-		if v.Value != gov.voteMap.GetValue(v.Key).Value {
-			t.Errorf("Encoded vote and Decoded vote are different! Encoded: %v, Decoded: %v\n", gov.voteMap.GetValue(v.Key).Value, v.Value)
-		}
-		gov.RemoveVote(v.Key, v.Value, 1000)
+		v, err := gov.ParseVoteValue(v)
+		assert.Equal(t, nil, err)
+		assert.Equal(t, gov.voteMap.GetValue(v.Key).Value, v.Value,
```

Comment:
Equal is enough? what about DeepEqual?

Reply:
Thanks! It's really helping comment.. Reflected at https://github.com/klaytn/klaytn/pull/1110/commits/70ad97142c82c7962c5150839e6fb074ae5091f3.